### DISCUSSION TITLE
SSD Replacement

### DISCUSSION THREAD

**Discussion by MakotoKumon** [HumanSupportRobot/QandA#88] [Q&A]
Is it possible to replace the internal SSD to a different device? We are currently trying to use Samsung SSD EVO870, but it makes the system non-bootable. If the EVO870 is connected to the board via SATA cable, the system halts even before showing BIOS screen.
Any information is helpful.
Thank you.
Makoto

**Comment by okdhryk** [comment]
I had a similar experience.
I replaced the SATA cable and it worked.
The SSD could be replaced with a new one.

**Reply by MakotoKumon** [reply]
Thank you for your information. Our group tested 6 different cables, but none of them worked.
We will purchase a different SSD drive to test.

**Comment by MakotoKumon** [comment]
After replacing the drive with a brand new Western Digital Blue SATA SSD (WDS100T3B0A), we made sure that the new one worked properly. So this issue could be caused by SSD compatibility. As we do not have any means to investigate further, let me close this topic.

Thank you!

Makoto
